src/parameterizations/gravity_wave_drag/orographic_gravity_wave_helper.jl

```diff
@@ -1,7 +1,212 @@
 using NCDatasets
 using Interpolations
 
-function get_OGW_info(Y, orographic_info_rll)
+include(joinpath(pkgdir(ClimaAtmos), "artifacts", "artifact_funcs.jl"))
```

Comment:
Can we fix the code loading here? This file is being repeatedly included in `src/` and I've been seeing a lot of gravity wave-related errors in CI lately (possibly related to this)